modules/salt/pages/formula-saltboot.adoc

```diff
@@ -120,7 +120,7 @@ In this example, SATA disks are differentiated from USB disks:
 /dev/disk/by-path/*usb*
 ----
 
-If the entered value does not contain "[systemitem]``/``", the entered value is automatically prepended by "[path]``/dev/disk/by-path/``".
+If the entered value does not contain "[systemitem]``/``, the entered value is automatically prepended by "[path]``/dev/disk/by-path/``."
```

Comment:
I do not understand why we want to have the double quotes here.

Reply:
I edited it to make vale happy, but you're right, they probably should go entirely.